### DISCUSSION TITLE
金錢還是名垂青史

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2275] [General]
我以前常跟朋友討論這件事情，你希望你有超多錢，還是沒錢但是你名垂青史？

**Comment by Teddy111802518** [comment]
我個人選擇名垂青史啦 錢死掉之後也用不到 名垂青史的話，我個人認為只要有人記得你 就不算死 真的死是當沒人記得你存在的時候 所以我成為一名研究者可學家 我希望我的成就能被後世傳唱 成為不死的

**Comment by christaaah** [comment]
我的話絕對是金錢，因為對我實質上有益，雖然這樣可能有點自私，但是這樣可以在我的人生當中就享受到，而且我也不想變的有名，名留青史的話，反正我死了，我也不知道死後的情況是怎樣，所以還是先選擇我知道的就好。

**Comment by kai1660** [comment]
理性要選金錢，感性要選名流千史，修過康沛的紅樓夢就會知道甚麼都是假的，死了之後甚麼都帶不走，所以我們應該要選錢，康沛給的分數也很糟糕，都是假的，推薦大家都去體驗看看。

**Comment by RoyLin1014** [comment]
我的目前的選擇是有錢，因為有名的話做什麼都會被放大檢視，我很不喜歡這種感覺。

**Comment by clairechien0627** [comment]
絕對是金錢，畢竟變有名實在沒有什麼好處，我也沒那麼大的分享欲，當個邊緣人生活挺好的，名垂青史的話感覺就會面對不知道是什麼鬼的東西，看現在的名人就知道了，不是被做成梗圖，就是被...???

**Comment by Teddy111802518** [comment]
謝謝大家的回覆啊 我覺得吧 我現在屬於 小孩子才做選擇我全都要 所以 我目前創業中 金錢名聲雙修 確實理性想大家會選擇錢啦 但是反正我本來就是個夢想家 你也可以認為我是個瘋子 所以我雖然想要很多錢 但是我果然還是想在短短的人生中為這個世界留下點什麼有意義的事物

**Comment by yuuu3957** [comment]
我選擇金錢，畢竟金錢還是比及實質且我能享受到的東西吧。
就算名垂青史，我死了後也不知道啊，而且看現在很多偉人都被拿來惡搞，我個人也是沒興趣被這麼做。
